.github/workflows/php-test.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           cache: npm
       - name: npm install
         run: npm ci
+      - name: Assets build
+        run: npm run build
```

Comment:
Oh, I believe this should also be added to the `deploy-dotork.yml` and `deploy-standalone-plugins.yml` GH workflows, right?

Reply:
Yes.